Data/_render_collections.qmd

```diff
@@ -280,6 +282,13 @@ for c in meta:
     except:
         frequency = ""
 
+    try:
+        Category = c["Category"]
+        
+    except:
+        Category = ""
```

Comment:
FYI: if `c` is a `dict`, you don't have to use these wasteful `try-except`  constructs, just do do a get with a fallback value:

```
category = c.get("Category", "")
```
side-notes:
- keep variables lower case: `category` instead of `Category`
- avoid catch-all `except:` clauses, at least use `except Exception:`

Reply:
Updated the variables naming. Also included get() in most of the cases. Include except Exception where needed. 

More improvements should be made in data_availability page.